backend/tests/BDD/step_defs/test_prompts.py

```diff
@@ -39,44 +39,48 @@ def check_response_includes_expected_response(context, prompt, expected_response
     response = send_prompt(prompt)
     actual_response = response.json()
 
-    try:
-        expected_value = Decimal(str(expected_response).strip())
-        actual_value = Decimal(str(actual_response).strip())
-
-        tolerance = Decimal("0.01")
-        is_equal = abs(expected_value - actual_value) <= tolerance
-
-        if not is_equal:
-            pytest.fail(f"\nNumeric values don't match!\n" f"Expected: {expected_value}\n" f"Actual: {actual_value}")
-
-    except (ValueError, decimal.InvalidOperation):
-        expected_str = str(expected_response).strip()
-        actual_str = str(actual_response).strip()
-
-        logger.info(f"Expected : {expected_str} \nActual: {actual_str}")
-
-        if actual_str.find(expected_str) == -1:
-            result = correctness_evaluator.evaluate_strings(
-                input=prompt,
-                prediction=expected_str,
-                reference=actual_str,
-            )
-
-            if result["value"] == "N":
-                logger.error(
-                    f"\nTest failed!\n"
-                    f"Expected: {expected_str}\n"
-                    f"Actual: {actual_str}\n"
-                    f"Reasoning: {result.get('reasoning', 'No reasoning provided')}"
-                )
-
-            assert result["value"] == "Y", (
+    # Allow `expected_response` to be a list of possible valid responses
+    possible_responses = [resp.strip() for resp in expected_response.split(",")]
```

Comment:
It's good that this now support multiple possible answers. 
I guess the implication of splitting on `,` and then doing a contains check for each substring though is that the resulting check is less strict. E.g. for a question like  
`Check the database and tell me the fund with the lowest Governance ESG score` with an expected response of 
`Dynamics Industries, Silvermans Global ETF, WhiteRocks ETF, which has a score of 60`.

This would pass for any answer that contains one of those fund names or which contains `which has a score of 60`.

Reply:
@mic-smith - I get your point, so what do you suggest we should do?

Reply:
I guess we could repeat the whole string in each option e.g. `Dynamics Industries which has a score of 60, Silvermans Global ETF which has a score of 60, WhiteRocks ETF which has a score of 60` maybe with a different separator than `,` so it's more obvious? Though not sure how well that would work with the case we need to use an LLM because the response isn't a substring.

We're also commenting these tests out in #23 . So, I wonder if it's worth pulling these test changes into in a separate branch so it doesn't block this PR and then addressing once we have the new flow to upload data and a different data set available?